docs/docs/user/hermes-mock.md

```diff
@@ -0,0 +1,105 @@
+# Hermes Mock
+
+A thin library designed to mocking Hermes in tests.
+
+## Overview
+
+Mocking is provided by `HermesMock` class, which consists of 3 parts:
+- `HermesMockDefine` 
+- `HermesMockExpect`
+- `HermesMockQuery`
+
+##### HermesMockDefine
+
+Is responsible for defining new topics on Hermes, consists of those methods:
+
+- `jsonTopic(String topicName)` - defines a JSON topic .
+
+- `avroTopic(String topicName)` - defines am Avro topic.
+
+- `jsonTopic(String topicName, int statusCode)` - defines a JSON topic with a specific response code.
```

Comment:
I'd rather say: `defines a JSON topic that when published on responds with given response code` or sth similar.